### DISCUSSION TITLE
Improvements to ballast flora

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#6394] [Ideas and suggestions]
Kinda disappointed by the new ballast flora, could've been better. It didn't really live up to my expectations, to any developer out there, I'd like to give you my ideas to make a update that makes the Ballast Flora better.

Inside joke aside, I came to the realization of why the ballast flora sucks- it's too easy to get rid of. The whole plant is literally irrelevant, because as soon as you kill the root, it all dissappears. So, to fix that, I have an idea. If a ballast flora captures a power item (batteries, jbox, supercap, etc) for 2-3 minutes, it becomes a new root! This sounds pretty overpowered, because say you had a box room. Now there's 30 roots to take care of 

If they were only allowed to become a root if there was a certain distance from the nearest one (e.g. more than 50 units) the root simply won't spawn

To anyone reading this, reply with your thoughts 

I'm curious to see if people would like it in the game

**Comment by mc-oofert** [comment]
I agree.

**Reply by StruggleDuck** [reply]
Yes

**Comment by trevormoney1** [comment]
I think that a different approach should be taken. This is quite an extreme buff to balast flora, as flora is already complicated to deal with for newer crew and usually results in many injuries when tackling it. Perhaps a better way to make balast flora more interesting to deal with would be any of the following.

A.  Different types of balast flora with different effects. May not fit lore.
B. Balast flora rips holes in submarine walls that lead to the outside of the submarine. This would make it much harder to deal with them as they would be constantly submerged in water.
C. When balast flora is killed, there is a chance of balast flora returning later at the same root site based on how big it got before being destroyed. This would result in the crew having to recheck the balast several times after it is killed, unless it was stifled early.

**Reply by girlyguppy** [reply]
Idk, I've had no issue with it at all from the start
The damage is weak, and really easy to avoid

Your last 2 suggestions would be good to add on to what I suggested, though 😎
